spec/hadoop_yarn_nodemanager_spec.rb

```diff
@@ -59,8 +59,8 @@
       end.converge(described_recipe)
     end
 
-    it 'ensures /usr/hdp/current/hadoop-yarn-nodemanager/bin/container-executor has proper permissions' do
-      expect(chef_run).to create_file('/usr/hdp/current/hadoop-yarn-nodemanager/bin/container-executor').with(
+    it 'ensures /usr/hdp/2.2.4.2-2/hadoop-yarn/bin/container-executor has proper permissions' do
```

Comment:
dumb question: why are we hardcoding the '2.2.4.2-2' portion of the path?
It seems like these could be break if the rev changes to 2.2.4.2-3, for instance (which would still match 2.2 and 2.2.4.2)